### PR TITLE
Fix typo for `JWTError.signatureVerificationFailed` case

### PR DESCRIPTION
**Pull request by davdroman**
Just something I noticed while using the library. Unless there's another way to do, this would be a breaking change, so if semver is to be strictly followed it'd involve a major version bump. Up to you :)

### CODE REVIEW

**Comment by 0xTim** [comment]
Oof yeah that's not something we can add it without a breaking change unfortunately. We'll need to save this for a major version bump

**Comment by 0xTim** [comment]
@ptoffy want to merge this into your branch?

**Comment by ptoffy** [comment]
> @ptoffy want to merge this into your branch?

This is fixed in my branch already 😅

File: Sources/JWTKit/JWTError.swift [hunk 1]
```diff
@@ -4,7 +4,7 @@ public enum JWTError: Error, CustomStringConvertible, LocalizedError {
     case claimVerificationFailure(name: String, reason: String)
     case signingAlgorithmFailure(Error)
     case malformedToken
-    case signatureVerifictionFailed
+    case signatureVerificationFailed
```
Comment (gwynne) [inline Sources/JWTKit/JWTError.swift:7]: This doesn't have to be a semver-major change. Just add this:
```swift
extension JWTError {
    @available(*, deprecated, renamed: "signatureVerificationFailed")
    public var signatureVerifictionFailed; JWTError { .signatureVerificationFailed }
}
```

Comment (0xTim) [reply Sources/JWTKit/JWTError.swift:7]: But @davdroman if you want to update it we can add it to v4

**Comment by davdroman** [comment]
Succeeded by #103.

**Pull request closed without merging**